src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const arrayOfValues = Object.values(robot);
+  const uniqueValues = new Set(arrayOfValues);
+
+  if (arrayOfValues.length !== uniqueValues.size) {
+    return null;
+  }
+
+  const arrayOfKeys = Object.keys(robot);
+  const inverseObj = {};
+
+  arrayOfKeys.forEach((key) => {
+    const newKey = robot[key];
+
+    inverseObj[newKey] = key;
+  });
```

Comment:
let's try to use `for in` loop to go through object

learn more about `for in`  - [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)

Reply:
thank you!